src/openms/include/OpenMS/METADATA/ExperimentalDesign.h

```diff
@@ -302,6 +316,7 @@ namespace OpenMS
 
       // Maps the Sample Entry to the row where the sample
       // appears in the Sample section
+      // TODO WHY is this needed?
```

Comment:
if not needed... remove.

Reply:
Would be a large refactor. Currently don't have the time